Winleafs.External/ReleaseClient.cs

```diff
@@ -7,21 +7,23 @@ namespace Winleafs.External
 {
     public class ReleaseClient : IReleaseClient
     {
-        private const string USERNAME = "StijnOostdam";
-        private const string REPOSITORY_NAME = "Winleafs";
+        private const string Username = "StijnOostdam";
```

Comment:
Shouldn't the user name be `winleafs`?

Reply:
You can test this by changing the version number in `UserSettings`

Reply:
Tested it by simply just placing a breakpoint and yes, the API call succeeded just fine.